src/components/SubmitButton.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC } from 'react';
+import { Box, BoxProps, Button, ButtonProps, Flex, Spinner } from '@chakra-ui/react';
+import { motion } from 'framer-motion';
+import Image from 'next/image';
+import { ImportantText } from './UI/headings';
+
+import { useShadowAnimation } from '../hooks';
+
+import planeVectorSVG from '../../public/images/plane-vector.svg';
+
+interface Props {
+  id?: string;
+  isValid: boolean;
+  isSubmitting: boolean;
+  isNewsletter?: boolean;
+  height: string;
+  width: string;
+  text: string;
+}
+
+const MotionBox = motion<BoxProps>(Box);
+const MotionButton = motion<ButtonProps>(Button);
+
+export const SubmitButton: FC<Props> = ({
+  id = 'submit-application',
+  isValid,
+  isSubmitting,
+  isNewsletter = false,
+  height,
+  width,
+  text
+}) => {
+  const { shadowBoxControl, setButtonHovered } = useShadowAnimation();
+
+  return (
+    <Box id={id} position='relative'>
+      <MotionBox
+        backgroundColor='brand.button.shadow'
+        h={height}
+        w={width}
+        position='absolute'
+        animate={shadowBoxControl}
+        opacity={!isValid ? 0 : 1}
+      />
+
+      <MotionButton
```

Comment:
I thought you were going to use https://chakra-ui.com/docs/components/form/button#button-loading-state
or does it make things harder?

Reply:
It doesn't allow to apply custom styles, e.g.: it resets the font family